R/expectation.R

```diff
@@ -98,6 +98,24 @@ as_label_flt  <- function(quo) {
     paste0("`", quo_lab, "`")
 }
 
+# check for data frames provided as vars ----
+
+check_expect_data_pipe <- function(first_arg) {
+  if (as_label(first_arg) == ".")
+    stop(
+      "Your `vars` argument is a `.`. ",
```

Comment:
Hm, it's not always `vars`. For `expect_cond()` it's `cond1` while for `expect_base()` it's `var`

Reply:
Good call, have updated